infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/workflows/AddNodeWorkflow.java

```diff
@@ -55,7 +55,7 @@
     final UUID id;
 
     @Getter
-    final List<Action> actions;
+    protected List<Action> actions;
```

Comment:
why not keeping it package private?

Reply:
I know that L46 (protected Layout newLayout) is outside the scope of your change but even that can be package private instead of protected. If you're fine with limiting the visibilities, can you change that too?

Reply:
HealNodeWorkflow extends this class and sets this variable. We can't do that if this is private.

Reply:
I was suggesting "package private" or default (i.e. no modifier) not "private". so all the classes in the package have access which is the case here.

Reply:
actions should be instantiated when the workflow is created, why does it need to be changed to protected ?